packages/components/src/components/@shared/_kol-table-stateless-mixin.scss

```diff
@@ -30,11 +31,16 @@
 			outline: 0 !important;
 		}
 
+		.table:focus {
```

Comment:
Hi @anicyne, wenn wir das nur am Fokus machen, dann flackert das, ja.

Zudem bitte nicht im Basis-Style, sondern in jedem Theme individuell. Im Basis-Style machen wir kein Margin, Padding und Gap ... das kommt immer erst im Theme.

Hier bitte auch das unterste Beispiel mit `_pagination` versehen: https://release-2--kolibri-public-ui.netlify.app/#/table/horizontal-scrollbar